string robot_name
float64[] robot_state
